### DISCUSSION TITLE
Why there is no sample for Amazon.Lambda.SimpleEmailEvents ?

### DISCUSSION THREAD

**Discussion by robertorealdesk** [aws/aws-lambda-dotnet#1647] [Q&A] [answered]
Why there is no sample for Amazon.Lambda.SimpleEmailEvents ?

**Comment by ashishdhingra** [comment, marked as answer]
@robertorealdesk Good morning.
- For sample code, please refer https://github.com/aws/aws-lambda-dotnet/tree/master/Libraries/src/Amazon.Lambda.SimpleEmailEvents. 
- For event structure, please refer [Using AWS Lambda with Amazon SES](https://docs.aws.amazon.com/lambda/latest/dg/services-ses.html)
- For additional details, please refer [SES: Invoke Lambda function action](https://docs.aws.amazon.com/ses/latest/dg/receiving-email-action-lambda.html)

Thanks,
Ashish
